blocks/Img/ImgBase.js

```diff
@@ -254,6 +258,10 @@ export class ImgBase extends BaseComponent {
     return [...srcset].join();
   }
 
+  getSrc() {
+    return this._getUrlBase();
```

Comment:
Is it original image? Maybe add some basic operations like `/format/auto/-/quality/smart/`